front/app/modules/commercial/custom_idea_statuses/admin/containers/index.tsx

```diff
@@ -145,31 +154,28 @@ const IdeaStatuses = () => {
             <T value={defaultStatus.attributes.title_multiloc} />
           </FlexTextCell>
           <Buttons>
-            <Tooltip
-              placement="top"
-              theme="light"
-              disabled={false}
-              content={
-                <FormattedMessage
-                  {...messages.defaultStatusDeleteButtonTooltip}
-                />
+            <DeleteStatusButton
+              tooltipDisabled={false}
+              buttonDisabled
```

Comment:
This is a dummy button. This status can never be deleted, so it's always disabled.